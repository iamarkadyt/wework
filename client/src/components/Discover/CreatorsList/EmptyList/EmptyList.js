import React from 'react'
import './EmptyList.scss'

const EmptyList = () => (
  <p className="EmptyList__message">Nothing for you right now! Come back later!</p>
)

export default EmptyList
