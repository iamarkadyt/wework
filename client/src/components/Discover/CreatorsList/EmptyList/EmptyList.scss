@import 'variables';

.EmptyList__message {
    background-color: $color-bg-main;
    color: $color-font-secondary;
    padding: .3rem 1rem;
    margin-top: .7rem;
    border-radius: .1rem;
}
