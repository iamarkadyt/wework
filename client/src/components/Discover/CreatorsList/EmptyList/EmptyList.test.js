import React from 'react'
import EmptyList from './EmptyList'
import { shallow } from 'enzyme'

it('must match a snapshot', () => {
  const comp = shallow(<EmptyList />)
  expect(comp).toMatchSnapshot()
})
