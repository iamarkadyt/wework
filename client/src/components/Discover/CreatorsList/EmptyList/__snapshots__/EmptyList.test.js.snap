// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`must match a snapshot 1`] = `
<p
  className="EmptyList__message"
>
  Nothing for you right now! Come back later!
</p>
`;
